test/passing/ite_kr_closing.ml.ref

```diff
@@ -0,0 +1,182 @@
+let _ =
+  if b then
+    e
+  else (
+    e1 ; e2
+  )
+
+let _ =
+  if b then
+    e
+  else (
+    something loooooooooooooooooooooooooooooooong enough to_trigger a break ;
+    this is more
+  )
+
+let _ =
+  if b then (
+    e1 ; e2
+  ) else (
+    something loooooooooooooooooooooooooooooooong enough to_trigger a break ;
+    this is more
+  )
+
+let _ =
+  if b then (
+    something loooooooooooooooooooooooooooooooong enough to_trigger a break ;
+    this is more
+  ) else if b1 then (
+    something loooooooooooooooooooooooooooooooong enough to_trigger a break ;
+    this is more
+  ) else
+    e
+
+;;
+f
+  ( if loooooooooooooooooooooooooooooooooooooooooooooooooooooooooong then
+    ()
+  else
+    ()
```

Comment:
Should be indented by 2 ?

Reply:
No actually the indentation is consistent with what is already done, see Josh's answer: https://github.com/ocaml-ppx/ocamlformat/pull/603#discussion_r262851548

Reply:
Got it.